ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/worldstate/NodeDataRequest.java

```diff
@@ -132,10 +107,33 @@ public NodeDataRequest setRequiresPersisting(final boolean requiresPersisting) {
   }
 
   public final void persist(final WorldStateStorage.Updater updater) {
+    if (pendingChildren.get() > 0) {
+      if (totalPending.incrementAndGet() > max + 999) {
+        max = totalPending.get();
+        LOG.warn("Maximum pending nodes just reached {}", max);
```

Comment:
what we do in the case ?

Reply:
I am going to remove this log together with removal of the totalPending nodes.

Reply:
This is just a statistic that I use to make sure that the amount does not explode to be too high. It should get stable quickly after the fast sync gets initiated. A very high number or an always growing number would be a sign of a bug.